README.md

```diff
@@ -54,11 +54,11 @@ Copy and rename the file `env.json` using a text editor, file utility, or termin
 
 1. Next, add the desired settings for the application. 
     
-    1. Replace the provided `LOG_LEVEL` value with the minimum level for log messages you want to appear in output. `INFO` or `DEBUG` is recommended for most use cases; see Python's [logging](https://docs.python.org/3/library/logging.html) module.
-
     1. Under `MYSQL_DATABASE`, replace the empty strings and `0` with the parameters for the database you wish to connect to.
+    1. If you need a CA certificate to connect to the database specified in the `MYSQL_DATABASE` object, replace the `null` value for `SSL_CA_PATH` with the absolute path to your certificate.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `2`, was `1`](https://app.codacy.com/manual/ITS_Teaching_And_Learning/archive-db/pullRequest?prid=4978122)